### DISCUSSION TITLE
How should we call ourselves?

### DISCUSSION THREAD

**Discussion by damdo** [gokrazy/gokrazy#138] [Polls]
While I was writing the welcome message here on Discussions, earlier, I was trying to refer to the group of users/members of gokrazy and I was wondering how to call ourself.
For example:
- a Python user refers to themselves as a _pythonista_ (_pythonistas_ or _pythonists_ for plural).
- a Go user as _gopher_.
- a Kubernetes practitioner _kubernaut_.

What should we call ourselves as gokrazy users/developers/community members? :)

**Comment by stapelberg** [comment]
I’m curious to see how the poll plays out. I think I’d just write gokrazy users, or gokrazy folks :)

**Comment by dotvezz** [comment]
Gonuts?

Oh man I'll have to come up with more puns and come back later...

**Comment by geekgonecrazy** [comment]
Gokrazies?  Might not be taken well?  Seems too easy not to suggest though

**Comment by damdo** [comment]
It looks like @stapelberg got the most upvotes :) Are we happy to stick with gokrazy users/folks then? cc. @stapelberg

**Reply by stapelberg** [reply]
Yep, let’s stick with gokrazy users :)
